### ISSUE TITLE
superLU/superLU_dist symbols missing when using MPICH (PETSc 3.16 rebuild)

### ISSUE THREAD

**Issue by MarDiehl** [conda-forge/petsc-feedstock#125]
when building DAMASK (https://github.com/conda-forge/damask-feedstock), which relies on PETSc, I get errors of the type

```
undefined reference to `Destroy_LU'
```
full logs are available on https://dev.azure.com/conda-forge/84710dde-1620-425b-80d0-4cf5baca359d/_apis/build/builds/400217/logs/57 and https://dev.azure.com/conda-forge/84710dde-1620-425b-80d0-4cf5baca359d/_apis/build/builds/400217/logs/37

Note: It is not clear whether this is an superLU_dist (or even superLU) or a PETSc issue.

**Comment by MarDiehl** [comment]
the superLU_dist package is outdated (https://github.com/conda-forge/superlu_dist-feedstock/issues/31). Maybe updating the version in https://github.com/conda-forge/superlu_dist-feedstock will solve the issue.

**Comment by awvwgk** [comment]
Updated superLU_dist to 6.4.0 now, let me know if the issue is still present.

**Comment by MarDiehl** [comment]
> Updated superLU_dist to 6.4.0 now, let me know if the issue is still present.

thanks! I still have the issue of unresolved symbols, but this might be due to a missing rebuild of PETSc. Probably PETSc should be rebuild after and update to superLU etc.

**Comment by dalcinl** [comment]
@MarDiehl Can we close this issue?

**Comment by MarDiehl** [comment]
Yes, I did not encounter any problems last months when updating the DAMASK feedstock
